pkg/common/idhelpers.go

```diff
@@ -20,7 +20,8 @@ type (
 	}
 )
 
-const linodeBSLabelLength = 32
+// TODO: Rename this variable
+const linodeStorageLabelLength = 32
```

Comment:
might want to change this and other variables from storage to volume?

Reply:
Oh yup linodeVolumeLabelLenght is more fitting!